prac_07/project_management.py

```diff
@@ -149,8 +150,14 @@ def add_new_project(projects):
 def filter_projects_by_date(projects):
     """Display projects filtered by date."""
     filtered_projects = []
-    date_string = input("Show projects that start after date (dd/mm/yy): ")
-    date = datetime.datetime.strptime(date_string, "%d/%m/%Y").date()
+    is_valid_date = False
+    while not is_valid_date:
+        try:
+            date_string = input("Show projects that start after date (dd/mm/yy): ")
+            date = datetime.datetime.strptime(date_string, "%d/%m/%Y").date()
+            is_valid_date = True
+        except ValueError:
+            print("Enter valid date format (dd/mm/yy)")
```

Comment:
It's a good idea to handle the value error. Would it be even better if you refactor these lines into a separate function?